examples/__init__.py

```diff
@@ -1,4 +1,2 @@
 # Copyright 2022 MosaicML Examples authors
 # SPDX-License-Identifier: Apache-2.0
-
-__version__ = '0.0.4'
```

Comment:
Should we leave versioning even if its not pip installable? We will still have "release" tags...

Reply:
I think deleting versioning is reasonable, but can keep release tags? Like, we definitely have to keep the old releases. Not sure we need to do new releases, except for maybe one more explaining the new structure? wdyt?